bcgov/biohubbc-platform
# BHBC-2238: See Related Datasets

# Overview
Allows the user to see and navigate to information related to a given dataset.

## Applicable Links

 - [Jira @BHBC-2238](https://quartech.atlassian.net/browse/BHBC-2238)
 - [Wireframes](https://preview.uxpin.com/b2be84efca658f9d07b74a9c8e377e75c4070e6f#/pages/161112036/simulate/sitemap?mode=i)

## Description of relevant changes

 - Adds a Related Datasets table to the dataset page
 - Adds endpoint, service method, and appropriate tests for retrieving related datasets
 - Adds MUI Alert and MUI Chip styles from SIMS
 - Minor tweaks to the Dataset Artifacts component
 - Changes typedef for EML source from `string` to `Record<string, unknown>`

## Testing Procedure
1. Create a project and survey in SIMS
2. Click **Submit Data**
3. Click **SELECT ALL** and then submit.
4. In BioHub, navigate to the survey you submitted: **Map Search** -> **Species Inventory Project** -> **Find Data**

You should be able to:
 - [ ] See the project as a related dataset
 - [ ] Click on the project name and navigate to the related dataset page. **Since project submission is not implemented (and therefore the project EML has not been injested), you shoud expect to see an error on the project dataset page.**
